### PR TITLE
 MY-PORTFOLIO: Add Contact form.

### PR DESCRIPTION
**Pull request by Festy1**
In this pull request, the following changes were made:

- the background colour was corrected as requested by the reviewer.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -388,3 +391,86 @@ ul h3 {
   line-height: 24px;
   color: #000;
 }
+
+.footer {
+  background-image: url(./Images/conform.svg), url('images/Contact-me.png');
```
Comment (amejid) [inline style.css]: - [ ] Good job implementing the validations. One of the objectives of this project is to understand how to parse a design template precisely. Kindly update the path of your background image to match the template. 😎️

**Suggested Change**
```
background-image: url('./Images/conform.svg'), url('./Images/Contact-me.png');
```

Currently, it's looking as follows.

**Review by amejid — CHANGES_REQUESTED** [review]
Hi team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Linter checks passing :heavy_check_mark: 
- Formspree service integrated successfully :heavy_check_mark: 
- Good UI design :100: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -217,10 +217,34 @@ <h3 class="Frameworks">Skills</h3>
                     </li>
                 </ul>
             </div>
-
-
         </section>
     </section>
+    <!-- *****Contact me****** -->
+    <section class="footer">
```
Comment (KanzaTahreem) [inline index.html]: Excellent job adding the right path 👍 Your path is fixed now, but you still need to take a small step to reach the finish line 
- [ ] Please make sure that you follow the figma design as much as possible according to the [requiremnts](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md). Following is a quote for a related requirement 🔽 Please fix this by adding the correct background color to contact section. I know you can do that 💪

  > _You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)._

## Your design previously 👇
image

## Your design now 👇
image


## Figma design 👇
image

**Review by KanzaTahreem — CHANGES_REQUESTED** [review]
# Changes Required 🔁

Hi @Festy1 👋

Good job implementing the changes suggested by previous reviewer. However there is still a thing that needs to be fixed  🏁 But you are almost there!_

so_close

## Highlights 🎯

There are some excellent points that you have made perfectly. Keep up the great work! 👏
- ✅ Well done so far following the given Figma design
- ✅ Great job with CSS
- ✅ Correct Linters

However, there is always room for improvement.

## Required Changes 🛑

> _Please keep in mind that your long hours of work is appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄_

_Please check the comments under the review._

## Optional suggestions ❕
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Owusu-Desmond — APPROVED** [review]
Hi @Festy1 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 




## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the setup of the project ✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**